bindings_ffi/src/mls.rs

```diff
@@ -112,6 +115,96 @@ pub async fn connect_to_backend(
     Ok(Arc::new(XmtpApiClient(api_client)))
 }
 
+/**
+ * Static revoke a list of installations
+ */
+#[uniffi::export(async_runtime = "tokio")]
+pub async fn static_revoke_installations(
+    api: Arc<XmtpApiClient>,
+    db: String,
+    encryption_key: Option<Vec<u8>>,
+    inbox_id: &InboxId,
+    installation_ids: Vec<Vec<u8>>,
+) -> Result<Arc<FfiSignatureRequest>, GenericError> {
+    let api = ApiClientWrapper::new(Arc::new(api.0.clone()), strategies::exponential_cooldown());
+
+    let storage_option = StorageOption::Persistent(db);
+
+    let store = match encryption_key {
+        Some(key) => {
+            let key: EncryptionKey = key
+                .try_into()
+                .map_err(|_| "Malformed 32 byte encryption key".to_string())?;
+            let db = NativeDb::new(&storage_option, key)?;
+            EncryptedMessageStore::new(db)?
+        }
+        None => {
+            let db = NativeDb::new_unencrypted(&storage_option)?;
+            EncryptedMessageStore::new(db)?
+        }
+    };
```

Comment:
If someone blows up there local database they likely wont still have access to their encryption key. Can we still do this revoke on an ephemeral database? I tried that but it errored.